nautobot_chatops/tables.py

```diff
@@ -34,6 +35,7 @@ class CommandLogTable(BaseTable):
         template_code='<span class="label label-{{record.status_label_class}}">{{ record.status }}</span>'
     )
 
+    # pylint: disable=too-few-public-methods
```

Comment:
I'd be fine with disabling `too-few-public-methods` globally in `pyproject.toml` instead of inline here.

Reply:
Done